test/Ansatz/Chain_test.jl

```diff
@@ -147,6 +147,45 @@
             @test length(tensors(canonize_site(qtn, Site(2); direction = :left, method = :svd))) == 4
         end
 
+        @testset "canonize" begin
+            using Qrochet: isleftcanonical, isrightcanonical
+
+            qtn = Chain(State(), Open(), [rand(4, 4), rand(4, 4, 4), rand(4, 4, 4), rand(4, 4, 4), rand(4, 4)])
+            canonized = canonize(qtn)
+
+            @test length(tensors(canonized)) == 9 # 5 tensors + 4 singular values vectors
+            @test isapprox(
+                contract(transform(TensorNetwork(canonized), Tenet.HyperindConverter())),
```

Comment:
I think you no longer need to do the Tenet.HyperindConverter transformation to be able to contract. Would you mind updating your Tenet installation and trying it?

Reply:
I now tried using `Tenet v0.5.2` and it did not work without the `Tenet.HyperindConverter`.